src/main/java/com/hardvan/springboot/springolearn/controller/BookController.java

```diff
@@ -2,6 +2,7 @@
 
 import com.hardvan.springboot.springolearn.model.Book;
 import com.hardvan.springboot.springolearn.service.BookService;
+import com.hardvan.springboot.springolearn.service.BookFilter;
```

Comment:
```suggestion
import com.hardvan.springboot.springolearn.service.BookFilter;
import com.hardvan.springboot.springolearn.service.BookService;
```

The imports should be in ascending order.

Reply:
Okay. Why so?

Reply:
Just a java convention.